charts/opensearch/CHANGELOG.md

```diff
@@ -13,6 +13,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 ---
+## [2.14.2]
+### Added
+### Changed
+### Deprecated
+### Removed
+### Fixed
+- Allow Extra Objects to be created via structured data, preventing unstructured errors
+- Adds logic to prevent Out-Of-Sync status for deployments in ArgoCD
```

Comment:
Thought I would add this in since I'm submitting this PR, the same behavior can be seen at https://github.com/bitnami/charts/issues/15526